### DISCUSSION TITLE
I can't install fate_flow

### DISCUSSION THREAD

**Discussion by Mrtrieu69** [FederatedAI/FATE#5429] [Q&A]
Hi, I'm running the command: python -m pip install fate_client[fate,fate_flow]==2.0.0. But there was an error that fate_client does not have version 2.0.0. I tried running the command like this: python -m pip install fate_client[fate,fate_flow], but the fate_flow package is not installed. 

Please tell me how I can install fate_flow.

**Comment by dylan-fan** [comment]
Now, it's ok
